File: internal/content/testdata/bench/go/coin-change.go
func coin_change(coins []int, amount int) int {
	INF := amount + 1
	dp := make([]int, amount+1)
	for i := 1; i <= amount; i++ {
		dp[i] = INF
	}
	for a := 1; a <= amount; a++ {
		for _, c := range coins {
			if c <= a && dp[a-c]+1 < dp[a] {
				dp[a] = dp[a-c] + 1
			}
		}
	}
	if dp[amount] == INF {
		return -1
	}
	return dp[amount]
}

File: internal/content/testdata/bench/csharp/coin-change.cs
public class Solution {
    public long coin_change(long[] coins, long amount) {
        long INF = amount + 1;
        var dp = new long[amount + 1];
        for (int i = 1; i <= amount; i++) dp[i] = INF;
        for (long a = 1; a <= amount; a++) {
            foreach (long c in coins) {
                if (c <= a) dp[a] = Math.Min(dp[a], dp[a - c] + 1);
            }
        }
        return dp[amount] != INF ? dp[amount] : -1;
    }
}
